config/routes.rb

```diff
@@ -3,6 +3,13 @@
   get '/login', to: 'login#index'
   get 'auth/google', as: :google_connect
   get 'auth/google/callback',  to: 'sessions#create'
+
   get '/restaurant', to: 'restaurants#show', as: :restaurant
+  get '/refine', to: 'refine#show', as: :refine
```

Comment:
I don't think we need to specify an alias if it's called the same as the path, Rails magic!